vcstool/commands/custom.py

```diff
@@ -72,6 +72,17 @@ def main(args=None):
     output_results(results, hide_empty=args.hide_empty)
     return 0
 
+def git_main(args=None):
+    return main(['--git','--args']+(args if args else sys.argv[1:]))
```

Comment:
The condition is to unspecific. If the function gets invoked e.g. with an empty list it falls back to sys.argv. Please compare with `is not None`.

For readability can you also please move setting the args before the return:

    if args is None:
        args = sys.argv[1:]

Please add a space after the comma and around the plus operator (PEP8) - same below.